### PR TITLE
Merge Staging Changes into Production

### PR DESCRIPTION
**Pull request by ccrawford4**

### CODE REVIEW

File: src/.example.env [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 # For deployment behind reverse proxy
 APP_BASE_URL=
+PROTOCOL='http'
 
 # humanID Core
 HUMANID_URL=
```
Comment (ccrawford4) [inline src/.example.env:10]: I noticed this .env is placed inside the src directory. However, upon checking the .env in the production web-login I found that there is no .env inside the src directory - only the root. Should I create this .env and place these values into it?

Comment (kofimokome) [reply src/.example.env:10]: put it here `/opt/humanid/.web-login.env`

Comment (kofimokome) [reply src/.example.env:10]: the value should be `https` on the server.

Comment (ccrawford4) [reply src/.example.env:10]: great thank you!

**Review by kofimokome — APPROVED** [review]

**Pull request merged**